test/callbacks.jl

```diff
@@ -466,6 +478,13 @@ end
           tstops=[5.1]
           test_discrete_callback(cb,tstops,g,dg!,cboop)
         end
+        @testset "tprev dependent callback" begin
+          condition(u,t,integrator) = t == 5
+          affect!(integrator) = (@show integrator.tprev; integrator.u[1] += integrator.t-integrator.tprev)
+          cb = DiscreteCallback(condition,affect!)
+          tstops = [4.999, 5.0]
```

Comment:
The first `tstop` value here forces the integrator to step to 4.999. This allows for a unified stepping behavior.